src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="search-form search-form--big"
+      data-qa="big"
+    >
+      <label class="search-form__label">
+        <span class="search-form__image search-form__image--big"></span>
```

Comment:
Just set all styles in this selectors to search-form__input, you don`t need this additional span here and in the small form